x-pack/legacy/plugins/siem/server/lib/detection_engine/rules/patch_rules.ts

```diff
@@ -76,7 +77,6 @@ export const patchRules = async ({
     type,
     references,
     version,
-    throttle,
```

Comment:
This means that changes to `throttle` (and `actions`) will not bump the rule version, right? Is that the correct behavior?

Reply:
After discussion with @rylnd and @FrankHassanabad we have decided to bump the rule's version whenever `actions` or `throttle` change